src/components/SearchBox.css

```diff
@@ -0,0 +1,39 @@
+.searchSection {
+  background-color: var(--tangerine);
+  width: 100%;
+}
+
+.searchBar {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  line-height: 1.5em;
+  padding: 2% 7% 0 7%;
```

Comment:
This is interesting to me. I’m used to seeing either `px` or `rem` units for padding. Is there a reason that `%` worked better here?